session/session.go

```diff
@@ -235,13 +235,16 @@ func (s *session) OpenCh(
 	}
 	partAddrs := []pwallet.Address{s.user.OffChainAddr, peer.OffChainAddr}
 	parts := []string{perun.OwnAlias, peer.Alias}
-	proposal := pclient.NewChannelProposal(
-		challengeDurSecs,
-		s.user.OffChainAddr,
-		allocations,
-		partAddrs,
-		pclient.WithApp(app.Def, app.Data),
-		pclient.WithRandomNonce())
+	proposal := &pclient.LedgerChannelProposal{ // Make proposal.
```

Comment:
Is this change required ? Is it not possible to use the `pclient.NewChannelProposal` function as it was before ?

Reply:
[Updated](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c)

Reply:
Can you move that change from [add transactor interface](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c) commit to this one, as it is a part of this change set ?

Reply:
[Moved](https://github.com/hyperledger-labs/perun-node/pull/110/commits/03c61bdc7d403de9979c2b71a061e4521870b65a)